src/BloomExe/Edit/EditingView.cs

```diff
@@ -522,14 +520,9 @@ private void _browser1_OnBrowserClick(object sender, EventArgs e)
 					ge.Handled = false; //let gecko handle it
 					return;
 				}
-				else
-				{
-					ErrorReport.NotifyUserOfProblem("Bloom did not understand this link: " + anchor.Href);
-					ge.Handled = true;
-				}
-
+				ErrorReport.NotifyUserOfProblem("Bloom did not understand this link: " + anchor.Href);
```

Comment:
JT already merged this PR, but for the future, I'd say the origin was more clear.